docs/options api.md

```diff
@@ -86,17 +86,29 @@ class ExampleWorld(World):
 ```
 
 ### Option Groups
-Options may be categorized into groups for display on the WebHost. Option groups are displayed alphabetically on the
-player-options and weighted-options pages. Options without a group name are categorized into a generic "Game Options"
-group.
+Options may be categorized into groups for display on the WebHost. Option groups are displayed in the order as specified
+by your world on the player-options and weighted-options pages. In the generated template files, there will be a comment
+with the group name at the beginning of each group of options. The `start_collapsed` only affects how the groups appear
+on the WebHost, with the grouping being collapsed when this is `True`.
+
+Options without a group name are categorized into a generic "Game Options" group, which is always the first group. If
+every option for your world is in a group, this group will be removed. There is also an "Items & Locations Options"
+group, which is automatically created using certain specified`item_and_loc_options`. These specified options cannot be
+removed from this group.
+
+Both the "Game Options" and "Item & Location Options" groups can be overridden by creating your own groups with
+those names, and specify specific options to add to them, as well as change the visibility. The "Item & Location
```

Comment:
```suggestion
those names, and specifying specific options to add to them, as well as change the visibility. The "Item & Location
```
I'm not entirely sure what "as well as change the visibility" is meant to refer to here. Is it the visibility of the options that are in those groups by default?

Reply:
https://github.com/ArchipelagoMW/Archipelago/pull/3370

Reply:
so i suppose that should be `start_collapsed`?